.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+ - '3.6'
+install:
+ - pip install -r requirements.txt
+script:
+ - pytest --suppress-no-test-exit-code
```

Comment:
Make sure this suppress flag is only temporary. Don't want this in the whole time.